build.go

```diff
@@ -122,6 +122,7 @@ type Builder interface {
 func Build(builder Builder, options ...Option) {
 	config := Config{
 		arguments:         os.Args,
+		bomLabel:          false,
```

Comment:
Should we instead just call this `allowDeprecated` ?

Reply:
IMHO, that's not specific enough. There could be other deprecated things to allow.

I'd be OK with `allowDeprecatedBOMLabel` and `WithDeprecatedBOMLabel`. I can make that change and resubmit in the morning if you're Ok with that.

For what it's worth, I did flag `WithBOMLabel` as deprecated, so if it's used it'll show up as deprecated in your IDE. Is that sufficient? or would you still prefer it in the variable name?